cron/monthly/invoice-platform-fees.js

```diff
@@ -236,9 +236,9 @@ export async function run() {
       FROM
         "Transactions" t
       LEFT JOIN "Transactions" ot ON
-        t."PlatformTipForTransactionGroup"::uuid = ot."TransactionGroup"
+        t."TransactionGroup" = ot."TransactionGroup"
         AND ot.type = 'CREDIT'
-        AND ot."PlatformTipForTransactionGroup" IS NULL
+        AND ot.kind IN ('CONTRIBUTION', 'ADDED_FUNDS') -- we only support adding tips on contributions and addedd funds for now
```

Comment:
Included `ADDED_FUNDS` in prevision for https://github.com/opencollective/opencollective/issues/3652